apps/wing/src/wingc.ts

```diff
@@ -108,14 +108,21 @@ export async function load(options: WingCompilerLoadOptions) {
   } else {
     // mapping the root is not sufficient on linux/mac
     // we need to also map all directories in the root
+    const rootStat = await fs.promises.stat("/");
```

Comment:
nit: to avoid toString later

```suggestion
    const rootDeviceId = await fs.promises.stat("/").then((stat) => stat.dev.toString);
```